crawl-ref/source/dat/des/altar/okawaru_arena.des

```diff
@@ -3,7 +3,7 @@
 
 # Typo below
 {{
-function callback.oka_start_challange(data, triggerable, triggerer, marker, ev)
+function callback.oka_start_challenge(data, triggerable, triggerer, marker, ev)
```

Comment:
Changing this will break save compat for existing games, I believe. That's why 335d1b35713 added a comment without changing it.